src/connections/allContacts.js

```diff
@@ -38,6 +38,11 @@ export const importContact = async (client, attributes) => {
   return client.create(DOCTYPE_CONTACTS, attributes, null)
 }
 
+export const updateContact = ({ client, contact, attributes }) => {
+  const updatedContact = merge({}, contact, attributes)
+  return client.save(updatedContact)
+}
```

Comment:
nit: 

I saw this in the same file, maybe it does the same thing ? 
```
const updatedContact = mergeContact(contacts[0], attributes)
    return client.save(updatedContact)
```

Reply:
Bien vu, en effet toute une logique a été faite autour de ça, autant l'utiliser 👍